documentation/book/proc-adding-volumes-to-jbod-storage.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// assembly-storage.adoc
+
+[id='proc-adding-volumes-to-jbod-storage-{context}']
+= Adding volumes to JBOD storage
+
+This procedure describes how to add volumes to a Kafka cluster configured to use JBOD storage.
+It cannot be applied to Kafka clusters configured to use any other storage type.
+While adding new volumes, you should not change configuration of any other volumes or remove them.
```

Comment:
should be it a WARNING to be highlighted somehow?

Reply:
It is not something what would have fatal consequences ... just the CO would not do anything. So I'm not sure it should be a warning. Also, in the second procedure about removing volumes, there is already an `IMPORTANT` not about loosing messages. I'm afraid that using `WARNING` for this would make that less visible.